plugins/sudoers/timestamp.c

```diff
@@ -959,7 +959,7 @@ bool
 timestamp_update(void *vcookie, struct passwd *pw)
 {
     struct ts_cookie *cookie = vcookie;
-    bool ret = false;
```

Comment:
Your change to make `ret` bool is correct, no need to revert that here.

Reply:
You were right. I was negligent

Let me resubmit it